roles/keystone/tasks/main.yml

```diff
@@ -31,6 +31,15 @@
   notify:
     - restart keystone services
 
+- name: install ldap pip reqs in package venv
```

Comment:
I see gift wrap is <i>  "A tool for creating bespoke system-native OpenStack artifacts."</i> 
python-ldap and ldappool are python libraries that the ldap driver is dependent on, they are not openstack projects.